chirp/drivers/tdh8.py

```diff
@@ -1540,6 +1556,65 @@ def _get_settings(self):
                                 current_index=_press.lsidekey4))
             basic.append(rs)
 
+        if self.MODEL in H3_LIST:
```

Comment:
This is really not the best way to do this, as I'm sure you know, but unfortunately it's a prevalent pattern throughout the code base. No need to change, just complaining.

Reply:
I tend to be a code chameleon — what pattern would you have preferred?